openedx_tagging/core/tagging/models/utils.py

```diff
@@ -34,21 +35,48 @@ def as_sqlite(self, compiler, connection, **extra_context):
         )
 
 
-class StringAgg(Aggregate):  # pylint: disable=abstract-method
+class StringAgg(Aggregate, Combinable):
     """
     Aggregate function that collects the values of some column across all rows,
-    and creates a string by concatenating those values, with "," as a separator.
+    and creates a string by concatenating those values, with a specified separator.
 
-    This is the same as Django's django.contrib.postgres.aggregates.StringAgg,
-    but this version works with MySQL and SQLite.
+    This version supports PostgreSQL (STRING_AGG), MySQL (GROUP_CONCAT), and SQLite.
     """
+    # Default function is for MySQL (GROUP_CONCAT)
     function = 'GROUP_CONCAT'
     template = '%(function)s(%(distinct)s%(expressions)s)'
 
-    def __init__(self, expression, distinct=False, **extra):
+    def __init__(self, expression, distinct=False, delimiter=',', **extra):
+        self.delimiter = delimiter
```

Comment:
Is this used, or is it always `,` ? Need to either update line 55 below or remove this and update the docstring.